Sources/NIOCore/AsyncAwaitSupport.swift

```diff
@@ -98,7 +98,7 @@ extension Channel {
     /// Get the value of `option` for this `Channel`.
     @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
     @inlinable
-    public func getOption<Option: ChannelOption>(_ option: Option) async throws -> Option.Value {
+    public func getOption<Option: ChannelOption>(_ option: Option) async throws -> Option.Value where Option.Value: Sendable {
```

Comment:
ChannelOption.Value should be `Sendable` but currently we can't do so in source stable way #2108